### ISSUE TITLE
A11y_VisualStudioCodeClient_ Customizations and Adding Extensions _ Adding Extension_Name Role Value: No Name is defined for settings button.

### ISSUE THREAD

**Issue by keerthiduvvuri** [microsoft/vscode#191069]
[[Check out Accessibility Insights!](https://accessibilityinsights.io/)]- Identify accessibility bugs before check-in and make bug fixing faster and easier.
### GitHubTags:
#A11yWCAG2.1; #A11yTCS; #A11ySev2; #Visual Studio Code Client; #BM-VisualStudioCodeClient-MAC-Aug2023; #Voiceover; #MAC; #WCAG4.1.2; #Name Role Value.
### Environment and OS details:
Visual Studio Code insider for MAC version 1.82.0
Mac Version: 13.5
Screen reader: Voice Over
### Repro Steps:
1.	Open Visual Studio Code Insiders editor
2.	Using "Ctrl+Shift+X" key navigate to the extension side view bar option.
3.	Using Tab key navigate to the "More" option and select different options(“Show installed Extension, Show outdated extension, Show popular Extension,Etc")
4.	Select an extension and using "Tab" and "Arrow" keys navigate through the extension details window on the RHS
5.	Using tab key navigate to different extension for "Installation/Uninstallation".
6.	Turn on Voice over and observe that No Name is defined for settings button.
### Actual Result:
No Name is defined for settings button.
### Expected Result:
No Name should be defined as settings button.
### User Impact:
Screen reader users will face difficulty if No Name is defined for settings button.
### Attachment:

https://github.com/microsoft/vscode/assets/98728920/db13893f-0407-41f7-b061-58f494788fff

**Comment by meganrogge** [comment]
Much simpler repro steps:
Focus an extension in the extensions view, tab to the settings button. 🐛 there's no aria label such as "settings"

![Screenshot 2023-08-23 at 8 45 50 AM](https://github.com/microsoft/vscode/assets/29464607/3efe7363-4137-487d-a6a0-078efc33f6d2)

**Comment by meganrogge** [comment]
This is fixed
